es6-arrow-functions/main.js

```diff
@@ -0,0 +1,46 @@
+const $jokeContainer = document.querySelector('#joke-container');
+const $flashImage = document.querySelector('#flash-image');
+const $jokeForm = document.querySelector('#joke-form');
+
+const jokester = {
+  tellJoke: function (setup, punchline) {
+    $jokeForm.classList.add('d-none');
+    const $introStatement = this.renderJokePhrase('Hey Flash...');
+    this.appendJokePhrase($introStatement);
+    setTimeout(() => {
+      const $jokeSetup = this.renderJokePhrase(setup);
+      this.appendJokePhrase($jokeSetup);
+      setTimeout(() => {
+        const $jokePunchline = this.renderJokePhrase(punchline);
+        this.appendJokePhrase($jokePunchline);
+        flash.laugh();
+      }, 2000);
+    }, 2000);
+  },
+  renderJokePhrase: phrase => {
```

Comment:
This can cause problems that may not be apparent in the exercise code. It should remain `function`. But, don't worry about it. It's arcane and obsolete with es6 classes.